keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -201,9 +201,13 @@ For custom resources:
 
 ### Graduation Criteria
 
-#### Alpha -> Beta Graduation
+#### Alpha
 
-- [ ] Collect user feedback on adding support of `--subresource` for `apply`
+- Add the `--subresource` flag to get, patch, edit and replace commands.
+
+#### Beta
+
+- Add the `--subresource` flag to the apply command.
```

Comment:
I'm not quite convinced we want to support `--subresource` in `apply` command, given its general purpose of being a declarative command, which we covered in this presentation https://youtu.be/eOxy1PS5TyQ?t=780.

Reply:
> I'm not quite convinced we want to support --subresource in apply command

It's also not straightforward to implement. Ref: https://github.com/kubernetes/enhancements/pull/2600#discussion_r612628956, https://github.com/kubernetes/enhancements/pull/2600#discussion_r620193062

Is it ok to move this KEP to beta without adding `--subresource` support for apply?

Although, then I'm not sure what "beta" or "GA" for this feature would even mean 🤔

Reply:
> > I'm not quite convinced we want to support --subresource in apply command
> 
> It's also not straightforward to implement. Ref: [#2600 (comment)](https://github.com/kubernetes/enhancements/pull/2600#discussion_r612628956), [#2600 (comment)](https://github.com/kubernetes/enhancements/pull/2600#discussion_r620193062)
> 
> Is it ok to move this KEP to beta without adding `--subresource` support for apply?

It's totally fine, just add a note, that due to additional complexity and the nature of `kubectl apply` command we've decided not to expand it with this flag. 

> Although, then I'm not sure what "beta" or "GA" for this feature would even mean thinking

You can promote to beta, w/o any test, which can be added during promotion, or after. For GA we need tests covering every are, ideally unit, integration (cmd) and e2e.

Reply:
Addressed feedback and updated to the new KEP template.